src/components/Map/RSVMap.tsx

```diff
@@ -56,6 +65,7 @@ export const RSVMap: React.VFC<Props> = ({ geometry }) => {
       )}
       onMouseEnter={() => setCursorStyle('pointer')}
       onMouseLeave={() => setCursorStyle('grab')}
+      attributionControl={false}
```

Comment:
Why disabled you the attribution?

Reply:
was just for taking the screenshot of the map. I will add it back